actions/new.action.ts

```diff
@@ -250,4 +254,17 @@ export const retrieveCols = () => {
   }
 };
 
+const fileExists = (path: string) => {
+  try{
+    fs.accessSync(path);
+    return true;
+  } catch (err: any) {
+    if (err.code === 'ENOENT') {
+      return false;
+    }
+
+    throw err;
+  }
+}
```

Comment:
```suggestion
};
```

Reply:
On a separate note, this whole gitignore problem is there as the npm does not copy .gitignore files to the distributions. Since we are using angular schematics which allows template prefix/suffix to be added which can be automatically removed  through the schematics build process. I have achieved the same for my collection 
Reference: [https://github.com/maxgaurav/nestjs-template-schematics](https://github.com/maxgaurav/nestjs-template-schematics)

Would it be okay that do the same for the nestjs schematics as well by opening a PR which can allow managing of .gitignore file through schematics.

Reply:
sure! I'll close the PR https://github.com/nestjs/nest-cli/pull/1649 then

Which means that we can close this PR as well, right? then after your fix on that schematics issue, we could open another one to drop the workaround implemented on this repo

Reply:
I think this would still be needed as other's who have made their custom collection may be thinking of using the default git ignore created from this cli action. Otherwise it needs to be documented that this behavior will go away and they should bring their own gitignore template through their schematics.

Reply:
Yes, makes sense. But form then the responsibility of having .gitingore file will be to the schematics, if not provided then gitingore will not be there in the project structure.

Reply:
Once I complete the functionality I will map the pull request here for refernce.